src/ios/CDVFileTransfer.m

```diff
@@ -552,6 +552,8 @@ - (void)connectionDidFinishLoading:(NSURLConnection*)connection
 
     if (self.direction == CDV_TRANSFER_UPLOAD) {
         uploadResponse = [[NSString alloc] initWithData:self.responseData encoding:NSUTF8StringEncoding];
+        if (uploadResponse == nil)
+            uploadResponse = [[NSString alloc] initWithData: self.responseData encoding:NSISOLatin1StringEncoding];
```

Comment:
Looks there is [no easy way](http://stackoverflow.com/questions/21488674/readtextasync-in-windows-store-app-with-multibyte-file-and-javascript) to fix that on Windows.